doc/source/whatsnew/v1.0.0.rst

```diff
@@ -205,6 +205,7 @@ Other enhancements
   (:meth:`~DataFrame.to_parquet` / :func:`read_parquet`) using the `'pyarrow'` engine
   now preserve those data types with pyarrow >= 1.0.0 (:issue:`20612`).
 - The ``partition_cols`` argument in :meth:`DataFrame.to_parquet` now accepts a string (:issue:`27117`)
+- Added new writer for exporting Stata dta files in version 118, ``StataWriter118``.  This format supports exporting strings containing Unicode characters (:issue:`23573`)
```

Comment:
do users need to know about StataWriter118 or can they just use read_stata/to_stata like before?

Reply:
to_stata exposes everything for writing (IIRC). The readers expose some metadata that isn't easy to express as a DataFrame.

Reply:
This is the same enhancement I added for StataWriter117.